cmake/sdlchecks.cmake

```diff
@@ -1021,7 +1021,7 @@ macro(CheckHIDAPI)
     if(SDL_HIDAPI_LIBUSB)
       set(HAVE_LIBUSB FALSE)
 
-      set(LibUSB_PKG_CONFIG_SPEC libusb-1.0)
+      set(LibUSB_PKG_CONFIG_SPEC libusb-1.0>=1.0.16)
```

Comment:
OK, this is good. I suggest that we apply this directly to SDL3, then rebase/squash this PR and get it in.

Reply:
Sounds good!